### PR TITLE
CRUD Implementation for Post model

### LINKED ISSUE #9: CRUD Implementation for Post model
## REQUIREMENTS / DESCRIPTION

- [x] Create first view for Posts
- [x] User can Create a new Post.
- [x] User can Read a Post.
- [x] User can Update a post.
- [x] User can Delete a post.

### PR DESCRIPTION
**Pull request by BlazeDex**
# Description

- [x] Create first view for Posts
- [x] User can Create/Save a new Post.
- [x] User can Read a Post.
- [x] User can Edit / Update a post.
- [x] User can Delete a post.

# Screenshots

## First view
imagen

## Create
imagen

## Read
imagen

## Edit
imagen

### CODE REVIEW

File: app/views/main/home.html.erb [hunk 1]
```diff
@@ -1,2 +1,29 @@
-<h1>Main#home</h1>
-<p>Find me in app/views/main/home.html.erb</p>
+<h1>Feed del usuario <%= current_user.id %></h1>
+<%= Date.today %>
+<p>Esto es un template para probar funcionalidades, el diseño se modificará después.</p>
```
Comment (MauricioNC) [inline app/views/main/home.html.erb]: Don't let this kind of code (p tag)

File: app/views/posts/edit.html.erb [hunk 2]
```diff
@@ -0,0 +1 @@
+<%= render 'form', event: @event, title: 'Edit post' %>
```
Comment (MauricioNC) [inline app/views/posts/edit.html.erb]: Add a new empty line here

File: app/views/posts/show.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Publicación de <%= current_user.email %></h1>
+<h2><%=@post.body%></h2>
```
Comment (MauricioNC) [inline app/views/posts/show.html.erb]: Add a new empty line here

File: app/views/posts/new.html.erb [hunk 4]
```diff
@@ -0,0 +1 @@
+<%= render 'form', event: @event, title: 'New post' %>
```
Comment (MauricioNC) [inline app/views/posts/new.html.erb]: Add a new empty line here

File: app/views/users/_user.json.jbuilder [hunk 5]
```diff
@@ -0,0 +1,2 @@
+json.extract! user, :id, :name, :lastName, :birthday, :weight, :height, :email, :created_at, :updated_at
+json.url user_url(user, format: :json)
```
Comment (MauricioNC) [inline app/views/users/_user.json.jbuilder]: If you don't need all the json responses, remove all the corresponding code and files

File: app/views/users/index.json.jbuilder [hunk 6]
```diff
@@ -0,0 +1 @@
+json.array! @users, partial: "users/user", as: :user
```
Comment (MauricioNC) [inline app/views/users/index.json.jbuilder]: Same here

File: app/controllers/users_controller.rb [hunk 7]
```diff
@@ -0,0 +1,71 @@
...
+=begin
+  def destroy
+    @user.destroy!
+
+    respond_to do |format|
+      format.html { redirect_to users_url, notice: "User was successfully destroyed." }
+      format.json { head :no_content }
+    end
+  end
+=end
```
Comment (juperameza) [inline app/controllers/users_controller.rb]: Don't leave commented code on github

File: app/views/users/show.json.jbuilder [hunk 8]
```diff
@@ -0,0 +1 @@
+json.partial! "users/user", user: @user
```
Comment (MauricioNC) [inline app/views/users/show.json.jbuilder]: Same here

File: app/views/posts/edit.html.erb (hunk 2 above)
Comment (juperameza) [inline app/views/posts/edit.html.erb]: Add a new line at the end of this file

File: app/views/posts/new.html.erb (hunk 4 above)
Comment (juperameza) [inline app/views/posts/new.html.erb]: same here new line

File: app/views/posts/show.html.erb (hunk 3 above)
Comment (juperameza) [inline app/views/posts/show.html.erb]: same here new line

File: app/controllers/posts_controller.rb [hunk 9]
```diff
@@ -0,0 +1,60 @@
...
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @post = current_user.posts.build(post_params)
+
+    respond_to do |format|
+      if @post.save
+        format.html { redirect_to root_path, notice: 'Post creado exitosamente.' }
```
Comment (MauricioNC) [inline app/controllers/posts_controller.rb]: Change this flash message to English

**Review by MauricioNC — CHANGES_REQUESTED** [review]

**Review by juperameza — CHANGES_REQUESTED** [review]

**Review by MauricioNC — APPROVED** [review]

**Review by juperameza — APPROVED** [review]

**Pull request merged**